import unittest
from fractions import Fraction

# from . my_sum import sums
import struct
print(help(struct))
# class TestSum(unittest.TestCase):
#
#     def test_list_int(self):
#         """
#         Проверить, что он может суммировать список целых чисел
#         """
#         data = [1, 2, 3]
#         result = sums(data)
#         self.assertEqual(result, 6),
#
#     def test_list_fraction(self):
#         """
#         Проверить, что он может суммировать список фракций
#         """
#         data = [Fraction(1, 4), Fraction(1, 4), Fraction(2, 5)]
#         result = sums(data)
#         self.assertEqual(result, Fraction(9, 10))
#
#     def test_bad_type(self):
#         """
#         Здесь мы заведомо указываем ошибку чтобы тест не останавливался
#         """
#         data = "banana"
#         with self.assertRaises(TypeError):
#             result = sums(data)
#
#
# if __name__ == '__main__':
#     unittest.main()
